apps/admission-webhook-k8s/mutating-webhook-config.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook
+  labels:
+    spiffe.io/webhook: "true"
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: admission-webhook-svc
+      namespace: nsm-system
+      path: /mutate
+      port: 443
+  failurePolicy: Fail
+  matchPolicy: Equivalent
+  name: nsm-admission-webhook-k8s-5dfd78487d-26n5b.networkservicemesh.io
+  namespaceSelector: {}
+  objectSelector: {}
+  reinvocationPolicy: Never
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - pods
+    scope: '*'
+  - apiGroups:
+    - apps
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - deployments
+    - statefulsets
+    - daemonsets
+    - replicasets
+    scope: '*'
```

Comment:
Should we consider adding a selector so that webhook matches only resources with `networkservicemesh.io` annotation?
Currently webhook works with all pods in the cluster which may potentially [cause issues](https://github.com/networkservicemesh/cmd-admission-webhook-k8s/issues/282) for resources that don't even want to use NSM.

Reply:
I haven't found any options for using annotation as a filtering parameter

Reply:
"scope" is not applicable according to this documentation: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-rules
And I I haven't found any suitable solutions with `namespaceSelector` or `objectSelector`